### PR TITLE
Divider block

### PR DESCRIPTION
**Pull request by cohencodes**
Old PR: #9

### CODE REVIEW

File: src/plugins/blocksmith/components/divider-block-modal/index.jsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+  constructor(props) {
+    super(props);
+    const { block } = props;
+
+    this.state = {
+      hasUpdatedDate: block.has('updated_date'),
+      text: block.get('text') || '',
+      strokeColor: block.get('stroke_color', 'primary'),
+      strokeStyle: block.get('stroke_style', 'solid'),
+      updatedDate: block.has('updated_date') ? moment(block.get('updated_date')) : moment(),
```
Comment (efjacobson) [inline src/plugins/blocksmith/components/divider-block-modal/index.jsx]: @cohencodes the updatedDate stuff changed recently, checking the box on your branch crashes the page. check how the other components are doing it now (not using moment)

Comment (cohencodes) [reply src/plugins/blocksmith/components/divider-block-modal/index.jsx]: @efjacobson fixed

**Comment by cohencodes** [comment]
@efjacobson didn't push my latest changes, should be good now

File: src/plugins/blocksmith/components/generic-block-placeholder/index.jsx [hunk 2]
```diff
@@ -163,14 +162,16 @@ class GenericBlockPlaceholder extends React.PureComponent {
...
           )}
         </>
+        { config.label && (
         <div
           className="placeholder-label-holder ml-2 mt-1"
           style={{ width: `calc(100% - ${labelOffset}px)` }}
         >
           <p className={classNames('label float-left mr-2', config.preview ? 'mt-2' : 'mt-1', config.iconGroup ? 'mb-0' : 'mb-1')}>
-            <i>{label || config.label}{title}</i>
+            <i>{config.label || config.label}{title}</i>
```
Comment (efjacobson) [inline src/plugins/blocksmith/components/generic-block-placeholder/index.jsx]: @cohencodes `||` is redundant here

File: src/plugins/sys/components/picklist-picker/index.jsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        </FormGroup>
+      );
+    }
+
+    return (
+      <FormGroup>
+        <Label>{label}</Label>
+        <Select
+          creatable={allowOther}
+          isDisabled={!isEditMode}
```
Comment (efjacobson) [inline src/plugins/sys/components/picklist-picker/index.jsx]: @cohencodes add `isClearable` here please

**Pull request merged**